src/main/java/picard/cmdline/CommandLineProgram.java

```diff
@@ -102,6 +106,12 @@
 
     @Option(doc="Google Genomics API client_secrets.json file path.", common = true)
     public String GA4GH_CLIENT_SECRETS="client_secrets.json";
+
+    @Option(shortName = "use_jdk_deflater", doc = "Use the JDK Deflater instead of the Intel Deflater for writing with BlockCompressedOutputStreams", common = true)
+    public Boolean USE_JDK_DEFLATER = false;
+
+    @Option(shortName = "use_jdk_inflater", doc = "Use the JDK Inflater instead of the Intel Inflater for reading with BlockGunzippers", common = true)
+    public Boolean USE_JDK_INFLATER = false;
```

Comment:
These argument docs should be targeted at end users rather than developers, and should not mention class names. Eg.,

```
"Use the JDK Deflater instead of the Intel Deflater for writing compressed output"
"Use the JDK Inflater instead of the Intel Inflater for reading compressed input"
```